docker-compose.yml

```diff
@@ -4,8 +4,8 @@ services:
   web:
     tty: true
     build: .
-    command: bash -c "yarn && yarn start"
+    command: 'yarn start'
     ports:
-      - '8080:8080'
+      - "8080:8080"
```

Comment:
Why change this to double quotes and above to single? Will one or the other not work?